src/views/InProgress.vue

```diff
@@ -832,6 +838,37 @@ export default defineComponent({
         logger.error('Failed to update order', err)
       }
     },
+    removeRejectedOrder(rejectedProductIds: any, rejectedItems: any) {
```

Comment:
I think we can take some alternate route here. On rejection we are refetching in progress orders. Rejected items won't be associated with any shipment. So if we could use that to filter out the rejected order(all items rejected) or the rejected items. This way we can keep the logic in the fetchInProgressOrdersAdditionalInformation action itself.